tests/check/check_signed_video_sign.c

```diff
@@ -648,18 +648,18 @@ START_TEST(two_completed_seis_pending)
   ck_assert_int_eq(sv_rc, SV_OK);
   sv_rc = signed_video_add_nalu_for_signing(sv, i_nalu_1->data, i_nalu_1->data_size);
   ck_assert_int_eq(sv_rc, SV_OK);
-  sv_rc = signed_video_add_nalu_for_signing(sv, i_nalu_2->data, i_nalu_2->data_size);
-  ck_assert_int_eq(sv_rc, SV_OK);
-
-  // Now 2 SEIs should be available. Get the first one.
+  // Get the first SEI.
   sv_rc = signed_video_get_sei(sv, NULL, &sei_size_1);
   ck_assert_int_eq(sv_rc, SV_OK);
   ck_assert(sei_size_1 != 0);
   uint8_t *sei_1 = malloc(sei_size_1);
   ck_assert_int_eq(sv_rc, SV_OK);
   sv_rc = signed_video_get_sei(sv, sei_1, &sei_size_1);
   ck_assert_int_eq(sv_rc, SV_OK);
-  // Now get the second one.
+
+  sv_rc = signed_video_add_nalu_for_signing(sv, i_nalu_2->data, i_nalu_2->data_size);
```

Comment:
This should still be at top. Otherwise there will not be a queue of signatures/SEIs